src/test/kotlin/the/coding/force/exploring_kotlin_coroutines/service/coroutine/DeleteDataServiceCoroutineTest.kt

```diff
@@ -0,0 +1,62 @@
+package the.coding.force.exploring_kotlin_coroutines.service.coroutine
+
+import io.mockk.coEvery
+import io.mockk.coVerify
+import io.mockk.impl.annotations.InjectMockKs
+import io.mockk.impl.annotations.MockK
+import io.mockk.junit5.MockKExtension
+import io.mockk.just
+import io.mockk.mockk
+import io.mockk.runs
+import kotlinx.coroutines.test.runTest
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+import org.junit.jupiter.api.extension.ExtendWith
+import the.coding.force.exploring_kotlin_coroutines.entities.DataEntity
+import the.coding.force.exploring_kotlin_coroutines.exception.DataNotFoundException
+import the.coding.force.exploring_kotlin_coroutines.repository.DataRepository
+import java.util.Optional
+import kotlin.test.assertEquals
+
+@ExtendWith(MockKExtension::class)
+class DeleteDataServiceCoroutineTest {
+    @MockK private lateinit var dataRepository: DataRepository
+    @InjectMockKs private lateinit var deleteDataServiceCoroutine: DeleteDataServiceCoroutine
+
+    @Test
+    fun `should delete data when ID exists`() = runTest {
+        // Arrange: Scenario config
```

Comment:
```suggestion
        // Arrange
```

Praticamente, só se coloca as 3 palavras mágicas
// Arrange
// Act
// Assert

ou seja, sem comentários

Reply:
Resolvido, tirei os comentários dos testes unitários e do teste de integração para o controlador update